db/models/Reaction.go

```diff
@@ -14,8 +21,9 @@ type Reaction struct {
 	CommentID uint64 `json:"comment_id"`
 	UserID    uint64 `json:"user_id"`
 
-	IsLIke    bool `json:"is_like"`
-	IsDislike bool `json:"is_dislike"`
+	ReactionType  ReactionType `json:"reaction"`
+	// IsLike    bool `json:"is_like"`
+	// IsDislike bool `json:"is_dislike"`
```

Comment:
We are not migrating to ReactionType. So revert these changes.
Also we do not leave some "debug" comments in production